File: spec/feature.rb/game_spec.rb
#   require 'game'

#   describe Game do

#     let(:player) {double :player}
#     let(:computer) {double :computer}

#     it 'can have a winner' do
#     allow(player).to receive(:choice).with("paper")
#     allow(computer).to receive(:random_shape).and_return "rock"
#     game = Game.new(player, computer)
#     expect(game.result).to eq "YOU WON"
#   end
# end

File: jasmine/spec/GameSpec.js
describe("Game", function() {

  beforeEach(function() {
    game = new Game();
    player = new Player();
    computer = new Computer();
  });

  it("should display the text to start a new game", function() {
    expect(game.gameStart()).toEqual('Rock, Paper or Scissors?');
  });

  it("should declare player the winner", function() {
    game.gameStart();
    player.makeTurn('Rock');
    var spy = spyOn(computer, 'makeTurn').and.returnValue('Scissors');
    computer.makeTurn();
    expect(game.outcome()).toEqual('Player wins!');
  });

});
